### ISSUE TITLE
Fatal Error on clikcing plugin settings in Elgg4x & php8x

### ISSUE THREAD

**Issue by daniyaree** [ColdTrick/advanced_notifications#8]

**Comment by jdalsem** [comment]
In Elgg 4 the advanced notifications plugin has no plugin settings. My guess is that you installed using the zip and did not remove the existing code... that would leave a plugin settings view available (which is not working). Please let me know if i am wrong

**Comment by daniyaree** [comment]
This is not very urgent or important that is why I did not provide the details. Basically I was trying to look for an site-admin option who can set Site Notifications OFF for say, all blog comments (just an example) as a default setting for all users. I did install the zip but I am not sure what "existing code" was to be removed, and from where. Here is the screenshot below (Elgg 4x and php 8x) but let this issue remain closed for now. Thanks.
![Screenshot 2021-09-28 at 16-46-15 Plugins My New Community](https://user-images.githubusercontent.com/43325988/135077088-169ebf33-f04d-4143-b5a9-80b000d49417.png)
